### ISSUE TITLE
warnings

### ISSUE THREAD

**Issue by samuelgarcia** [python-quantities/python-quantities#184]
I have this warning with actual quantities release.
`quantities-0.12.4-py3.8.egg/quantities/__init__.py:1: DeprecationWarning: invalid escape sequence \m`
and also this one
`quantities-0.12.4-py3.8.egg/quantities/markup.py:90: DeprecationWarning: invalid escape sequence \c`

@apdavison : maybe it could be good to make a release of quantities befire the neo release no ?

**Comment by drammock** [comment]
This is now breaking the documentation builds of MNE-Python:

```
Traceback (most recent call last):
  File "/home/circleci/project/examples/io/read_neo_format.py", line 15, in <module>
    import neo
  File "/home/circleci/.local/lib/python3.8/site-packages/neo/__init__.py", line 10, in <module>
    from neo.core import *
  File "/home/circleci/.local/lib/python3.8/site-packages/neo/core/__init__.py", line 33, in <module>
    from neo.core.block import Block
  File "/home/circleci/.local/lib/python3.8/site-packages/neo/core/block.py", line 12, in <module>
    from neo.core.container import Container, unique_objs
  File "/home/circleci/.local/lib/python3.8/site-packages/neo/core/container.py", line 10, in <module>
    from neo.core.spiketrain import SpikeTrain
  File "/home/circleci/.local/lib/python3.8/site-packages/neo/core/spiketrain.py", line 27, in <module>
    import quantities as pq
  File "/home/circleci/.local/lib/python3.8/site-packages/quantities/__init__.py", line 1
    """
    ^
SyntaxError: invalid escape sequence \m
```

I infer that the problem is this line, and a double-backslash would fix it:

https://github.com/python-quantities/python-quantities/blob/6ee561e918c00fec773481fa9a2c73e8e106f061/quantities/__init__.py#L223

Our CIs load `quantities` version 0.12.4 and `neo` version 0.10.0

If I submit a PR with that fix, will you cut a release please?

**Comment by apdavison** [comment]
released v0.12.5

**Comment by drammock** [comment]
thanks 🎉
